pubsub/tests/system.py

```diff
@@ -80,12 +80,40 @@ def test_publish_messages(publisher, topic_path, cleanup):
                 num=str(index),
             ),
         )
-
     for future in futures:
         result = future.result()
         assert isinstance(result, six.string_types)
 
 
+def test_publish_max_bytes(publisher, topic_path, cleanup):
```

Comment:
This doesn't need to be a system test, it can likely just be a unit test.

Reply:
you mean just delete it?